### ISSUE TITLE
Describe field is not show in command function

### ISSUE THREAD

**Issue by yhojann-cl** [yargs/yargs#2140]
By example:

```javascript
const yargs = require('yargs');

yargs(process.argv.slice(2))
    .command('$0 <foo>', 'Description', (yargs) => {
        yargs
        .positional('foo', { describe: 'Foo', type: 'string', demand: true })
        .option('bar', { alias: 'b', describe: 'Bar' })
    }).argv
```

The result is:

```bash
$ moduleBinary
moduleBinary <foo>

Description

Positionals:
  foo                                                     [cadena de caracteres]

Opciones:
  --help     Muestra ayuda                                            [booleano]
  --version  Muestra número de versión                                [booleano]

Hacen falta argumentos no-opcionales: Número recibido 0, necesita por lo menos 1
```

`foo` description is hidden and `bar` option is hidden, but when call `--help` option can see all texts:

```bash
$ moduleBinary --help
moduleBinary <foo>

Description

Positionals:
  foo  Foo                                                [cadena de caracteres]

Opciones:
      --help     Muestra ayuda                                        [booleano]
      --version  Muestra número de versión                            [booleano]
  -b, --bar      Bar
```

Version: `17.3.1`.

**Comment by jly36963** [comment]
I believe this problem is solved in yargs 17.4. Prior to that version, there was a bug with failed default commands and the resulting usage output.
[PR](https://github.com/yargs/yargs/pull/2105)

```js
const input = ''

yargs(input)
  .command({
    command: '$0 <arg1>',
    desc: 'default desc',
    builder: yargs => yargs
      .option('arg1', {
        type: 'string',
        desc: 'arg1 desc (default command)',
        demandOption: true
      })
      .option('arg2', {
        type: 'string',
        desc: 'arg2 desc (default command)'
      }),
    handler: argv => {
      console.log(argv)
    }
  })
  .strict()
  .parse()
```

Before the fix:

```
Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]
  --arg1                                                     [string] [required]
  --arg2                                                                [string]
 ```
 
 After the fix:
 
 ```
 Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]
  --arg1     arg1 desc (default command)                     [string] [required]
  --arg2     arg2 desc (default command)                                [string]
 ```

**Comment by shadowspawn** [comment]
This was likely an interaction with the default command, and has been fixed.
